crates/fuel-core/src/query/balance/asset_query.rs

```diff
@@ -1,4 +1,5 @@
 use crate::graphql_api::database::ReadView;
+use async_graphql::futures_util::TryStreamExt;
```

Comment:
Do we need to use the `async_graphql` version of this trait? Otherwise I'd suggest using the one from `futures` directly (which just re-exports the same trait from `futures_util`)

```suggestion
use futures::TryStreamExt;
```